lib/multi.c

```diff
@@ -1177,29 +1217,65 @@ static CURLMcode Curl_multi_wait(struct Curl_multi *multi,
       ufds[nfds].events |= POLLPRI;
     if(extra_fds[i].events & CURL_WAIT_POLLOUT)
       ufds[nfds].events |= POLLOUT;
+#endif
     ++nfds;
```

Comment:
Is increasing nfds here still required with the new approach? I don't think it is used with WinSock anymore, right?

Reply:
I think the nfds here can be moved into the 2nd part of the #else clause.

Reply:
Oh, was wrong here. nfds is used later on to check if there is actually something to wait for.